functions/v2/helloworld/hello_pubsub.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START functions_helloworld_pubsub_v2]
```

Comment:
issue: I believe this region tag is supposed to be functions_cloudevent_pubsub, based on https://github.com/GoogleCloudPlatform/python-docs-samples/pull/6530/files and https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/8f2378b80a8ae2ad887b07f402a76f060e17fd9e/functions/v2/index.js#L17

Reply:
The file structure and region tags in the `golang-samples` were different from the other repos.  I copied this sample as a starting point:  https://github.com/GoogleCloudPlatform/golang-samples/blob/master/functions/helloworld/hello_pubsub.go. -- which had the region tag that I just tacked `v2` to the end of

Reply:
we should follow the python/nodejs region tag style so it doesn't show up as missing for Go